darts/models/forecasting/pl_forecasting_module.py

```diff
@@ -76,7 +83,8 @@ def __init__(
         super().__init__()
 
         # save hyper parameters for saving/loading
-        self.save_hyperparameters()
+        # do not save type nn.Module params
+        self.save_hyperparameters(ignore=["loss_fn", "torch_metrics"])
```

Comment:
Pytorch Lightning gives a warning for saving nn.Modules because they are already saved as a part of the model. Adding the ignore prevents them from being saved twice.